stytch/shared/rbac_local.go

```diff
@@ -17,7 +17,7 @@ func PerformAuthorizationCheck(
 	}
 
 	if subjectOrgID != authorizationCheck.OrganizationID {
-		return stytcherror.NewClientLibraryError("Subject organization ID does not match ID from request")
+		return stytcherror.NewSessionAuthorizationTenancyError()
```

Comment:
[sand] bonus if we can take `subjectOrgID` and `authorizationCheck.OrganizationID` as params for the constructor to display a helpful error message

Reply:
[5c737e0](https://github.com/stytchauth/stytch-go/pull/159/commits/5c737e045a12826dcc65a62e80b8f4e6428b88ac)